qa/smoke-test-rank-eval-with-mustache/build.gradle

```diff
@@ -24,10 +24,4 @@ apply plugin: 'elasticsearch.rest-test'
 dependencies {
     testCompile project(path: ':modules:rank-eval', configuration: 'runtime')
     testCompile project(path: ':modules:lang-mustache', configuration: 'runtime')
-}
-
-/*
```

Comment:
++. This is closed. If you want to just push a commit that removes that comment to the repo without waiting on a PR that'd be awesome.